geopandas/tests/test_pandas_methods.py

```diff
@@ -529,13 +529,12 @@ def test_apply_no_geometry_result(df, crs):
     if crs:
         df = df.set_crs(crs)
     result = df.apply(lambda col: col.astype(str), axis=0)
-    # TODO this should actually not return a GeoDataFrame
-    assert isinstance(result, GeoDataFrame)
+    assert isinstance(result, pd.DataFrame)
```

Comment:
this actually gives true for both DataFrame and GeoDataFrame, so best to make this a strict type check?